pkg/filesystem/pool/simple_sparse_reader_at.go

```diff
@@ -0,0 +1,130 @@
+package pool
```

Comment:
I'm not really convinced we need to ship an implementation this complex. What's the thing bb-remote-execution actually needs? Is it just an empty file? If so, why don't we offer that?